### DISCUSSION TITLE
Toast message shifting my form

### DISCUSSION THREAD

**Discussion by ghost** [fomantic/Fomantic-UI#1363] [Modules (Modal, Dropdown, Popup, Calendar, Toast, ...)] [answered]
How can I prevent my form from shifting a few pixels down as soon as a toast message is triggered?

**Comment by lubber-de** [comment]
🤔 This should not happen... could you please provide a jsfiddle , so we can reproduce?

**Comment by ghost** [comment, marked as answer]
My mistake. I put the message all the way up in the body to fix the error.
